apps/xui-slack-help-bot/xui-slack-help-bot/xui-slack-help-bot.yaml

```
@@ -0,0 +1,33 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: xui-slack-help-bot
+  namespace: xui-slack-help-bot
+spec:
+  releaseName: xui-slack-help-bot
+  values:
+    replicas: 1
+    image: hmctspublic.azurecr.io/xui/slack-help-bot:prod-ee7658a-20230315123456 #{"$imagepolicy": "flux-system:xui-slack-help-bot"}
+    environment:
+      SLACK_REPORT_CHANNEL: exui-bot-temp
+      SLACK_REPORT_CHANNEL_ID: C04TMKC9FHD
+      JIRA_PROJECT: EXUI
+      JIRA_ISSUE_TYPE_ID: 10900
+      JIRA_ISSUE_TYPE_NAME: "Bug"
+    aadIdentityName: xui-slack-help-bot
+    keyVaults:
+      cftptl-intsvc:
+        excludeEnvironmentSuffix: true
+        secrets:
+          - xui-slack-bot-token
+          - xui-slack-app-token
+          - jira-username
+          - jira-api-token
+  chart:
+    spec:
+      chart: nodejs
+      version: 2.4.8
```

use the latest version please
```suggestion
      version: 2.4.14
```

---

Serves me right for copy/pasting.. done!